Description: Allow the timeout on mssql to be effective per query
Origin: http://svn.php.net/viewvc?view=revision&revision=298255
Last-Update: 2010-05-02

Index: php/ext/mssql/php_mssql.c
===================================================================
--- php.orig/ext/mssql/php_mssql.c
+++ php/ext/mssql/php_mssql.c
@@ -1343,6 +1343,7 @@ PHP_FUNCTION(mssql_query)
 	mssql_result *result;
 	int id = -1;
 
+	dbsettime(MS_SQL_G(timeout));
 	batchsize = MS_SQL_G(batchsize);
 
 	if (zend_parse_parameters(ZEND_NUM_ARGS() TSRMLS_CC, "s|rl", &query, &query_len, &mssql_link_index, &zbatchsize) == FAILURE) {
